nixos/lib/testing-python.nix

```diff
@@ -1,18 +1,26 @@
+args@
 { system
 , pkgs ? import ../.. { inherit system config; }
   # Use a minimal kernel?
 , minimal ? false
   # Ignored
 , config ? { }
   # !!! See comment about args in lib/modules.nix
-, specialArgs ? { }
+, specialArgs ? throw "legacy - do not use, see error below"
```

Comment:
The default is dead code, as it is not used in the file, but a default is needed for the custom error message below to apply.